keyboards/handwired/marauder/info.json

```diff
@@ -0,0 +1,108 @@
+{
+    "keyboard_name": "Minshara Marauder",
+    "url": "",
+    "maintainer": "qmk",

```

Comment:
These should be filled out too.

Reply:
Sorted with a link to the build project and filling in my name under maintainer, hope this is all right. Thank you for your patience!